### PR TITLE
Fine tune user-data.yaml example file

### PR DESCRIPTION
**Pull request by davidcassany**
Just few minor tweaks to better align the user-data.yaml example with the installer.

### CODE REVIEW

**Review by mudler — APPROVED** [review]

**Comment by davidcassany** [comment]
This example data worked fine out of the box in AWS with an instance of  4GB of memory

File: images/user-data.yaml [hunk 1]
```diff
@@ -6,23 +6,18 @@ stages:
          # It will partition a device including the given filesystem label or part label (filesystem label matches first)
          device:
            label: COS_RECOVERY
-         # Only last partition can be expanded
-         # expand_partition:
-         #   size: 4096
          add_partitions:
            - fsLabel: COS_STATE
-             size: 8192
+             # 15GB for COS_STATE, the disk should be at least of 16GB
```
Comment (Itxaka) [inline images/user-data.yaml:11]: will this be enough? on my tests the recovery partition was 2Gb so this should be at least 17Gb?

**Review by Itxaka — APPROVED** [review]

File: images/user-data.yaml (hunk 1 above)
Comment (davidcassany) [reply images/user-data.yaml:11]: Yes you are right, gonna write it should be bigger than 17Gb

**Review by Itxaka — APPROVED** [review]

**Comment by mudler** [comment]
No sense to waste CI clocks here, merging :+1:

**Pull request merged**